portal/src/graphql/adminapi/UsersList.tsx

```diff
@@ -411,8 +411,9 @@ const UsersList: React.VFC<UsersListProps> = function UsersList(props) {
         case "roles": {
           return renderRoleCell(item);
         }
-        default:
-          return renderDefaultCell(item);
+        default: {
+          return renderDefaultCell(item, column);
```

Comment:
If column cannot be omitted, change the signature of `renderDefaultCell` to `renderDefaultCell(item: UserListItem, column: IColumn | undefined)` to force the caller to pass column to the function.